client/src/scenes/challenge/evaluations/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import { FlexChild, FlexContainer } from '../../../components/flex';
+import { helpers } from '../../../data/challenge_evaluations';
+import { helpers as spotHelpers } from '../../../data/spot';
+import { fetch } from '../../../utils';
+import Evaluation from './components/evaluation';
+import SideNav from './components/side-nav';
+import SideNavItem from './components/side-nav-item';
+
+class Evaluations extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.setActiveChallengeTo = this.setActiveChallengeTo.bind(this);
+
+    let currentChallengeId = null;
+
+    if (props.challenges) {
+      const sortedChallenges = props.challenges.sort((a, b) =>
+        spotHelpers.compareSpots(a.spot, b.spot)
+      );
+
+      currentChallengeId = sortedChallenges[0] && sortedChallenges[0].id
+    }
+
+    this.state = {
+      currentChallengeId,
+    };
+  }
+
+  render() {
+    if (!this.props.challenges) {
```

Comment:
Didn't realize initially that it wasn't rendered until the data was available. I'll get rid of that!